Neos.ContentRepository.Export/src/ExportService.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Export;
+
+/*
+ * This file is part of the Neos.ContentRepository.LegacyNodeMigration package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+
+use League\Flysystem\Filesystem;
+use Neos\ContentRepository\Core\Projection\Workspace\WorkspaceFinder;
+use Neos\ContentRepository\Export\Processors\EventExportProcessor;
+use Neos\ContentRepository\Core\Factory\ContentRepositoryServiceInterface;
+use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamId;
+use Neos\EventStore\EventStoreInterface;
+
+class ExportService implements ContentRepositoryServiceInterface
+{
+
+    public function __construct(
+        private readonly Filesystem          $filesystem,
+        private readonly WorkspaceFinder     $workspaceFinder,
+        private readonly EventStoreInterface $eventStore,
+        private readonly ContentStreamId     $contentStreamIdentifier,
```

Comment:
Just a cosmetic consideration:
```suggestion
        private readonly Filesystem $filesystem,
        private readonly WorkspaceFinder $workspaceFinder,
        private readonly EventStoreInterface $eventStore,
        private readonly ContentStreamId $contentStreamIdentifier,
```
Though, not just cosmetic: When aligning parameters like this, adding a new param can lead to a multi-line diff which makes reviewing harder